e2e/fuzz/replay/replay_message_command.go

```diff
@@ -18,18 +17,20 @@ import (
 type ReplayMessageCommand struct {
 	UI cli.Ui
 
-	filePath string
+	messagesFilePath string
+	metaDataFilePath string
 }
 
 // Help implements the cli.Command interface
-func (fc *ReplayMessageCommand) Help() string {
+func (rmc *ReplayMessageCommand) Help() string {
 	return `Runs the message and timeouts replay for analysis and testing purposes based on provided .flow file.
 	
-	Usage: replay-messages -file={fullPathToFlowFile}
+	Usage: replay-messages -messagesFile={fullPathToFlowFile} -metaDataFile={fullPathToFlowFile}
```

Comment:
Is it required to have both? Could we merge both in the same file?

Reply:
We could. Although its kinda messy to unmarshal the objects, since actions and messages are two different structures, and json.unmarshall is really dumb about these things, it needs to know the exact type in which to unmarshal, so it becomes a little bit tricky when reading the file.

Reply:
ok, lets go then with the first option. Command will take folder of the two files as input